core/request.py

```diff
@@ -462,9 +462,12 @@ def findAndCall(self, path: str, *args, **kwargs) -> None:
                         kwargs[key] = [kwargs[key], value]
                 else:
                     kwargs[key] = value
-                stopCount -= 1
-                if not stopCount:  # We reached zero; maximum PostParamsCount exceeded
-                    raise errors.NotAcceptable()
+
+                count -= 1
+                if count == 0:  # We reached zero; maximum PostParamsCount exceeded
+                    raise errors.NotAcceptable(
+                        f"""Too many arguments supplied, exceeding maximum of {total} allowed arguments per request"""
```

Comment:
Can we here use conf["viur.maxPostParamsCount"] instead of "total" ?

Reply:
We use it, but by a reference ;-) I did it this way to shorten the line to meet PEP8. Also, we don't do a __getattr__-call twice this way.